packages/flutter/lib/src/widgets/media_query.dart

```diff
@@ -213,6 +215,55 @@ class MediaQueryData {
   ///   property and how it relates to [padding] and [viewInsets].
   final EdgeInsets viewPadding;
 
+  /// The parts of the display where the system may be detecting certain
+  /// gestures for system actions.
+  ///
+  /// These gestures are typically drags from the edge of the screen to
+  /// perform system actions, like returning to the previous screen or
+  /// opening the notification tray. Gestures reserved for system actions
+  /// may not be forwarded to the app when performed in the area specified
+  /// by these insets. Therefore, elements placed in these areas may not properly
+  /// respond to them.
+  ///
+  /// This property is currently only expected to be set to a non-default value
+  /// on Android starting with version Q.
+  ///
+  /// {@tool snippet --template=stateful_widget_material}
+  ///
+  /// When using Android Q with full gestural navigation, use
+  /// [MediaQuery.systemGestureInsets] with [Padding] to avoid overlapping
+  /// a [Slider] with system gesture navigation.
+  ///
+  /// ```dart
+  /// double _currentValue = 0.2;
+  ///
+  /// @override
+  /// Widget build(BuildContext context) {
+  ///   EdgeInsets systemGestureInsets = MediaQuery.of(context).systemGestureInsets;
+  ///   return Scaffold(
+  ///     appBar: AppBar(title: Text('Sample Gesture Nav')),
+  ///     body: Padding(
```

Comment:
Need a comment here to explain that, by default, the slider expands to fill the available width.

Not clear why we're padding top/bottom (we're assuming something about systemGestureInsets?).  Maybe: 
```
Container(
  alignment: Alignment.center,
  padding: systemGestureInsets, // only left,right padding is likely to be needed
  child: Slider...
),
```

Reply:
I mainly pad the top/bottom to make it look cleaner in the sample app. I'll just set top and bottom to 0 for the sake of the sample to avoid confusion, which seems like a worthy tradeoff